File: Module.Persist.TPM/Model/History/HistoricalAgeGroup.cs
using Core.History;
using Module.Persist.TPM.Model.TPM;

namespace Module.Persist.TPM.Model.History
{
    [AssociatedWith(typeof(AgeGroup))]
    public class HistoricalAgeGroup : BaseHistoricalEntity<System.Guid>
    {
        public string Name { get; set; }
    }
}

File: Module.Frontend.TPM/Client/model/tpm/agegroup/HistoricalAgeGroup.js
﻿Ext.define('App.model.tpm.agegroup.HistoricalAgeGroup', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'AgeGroup',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true },
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'Name', type: 'string', hidden: false, isDefault: true }
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalAgeGroups',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});
